src/symbol.c

```diff
@@ -571,13 +563,13 @@ mrb_init_symbol(mrb_state *mrb)
 {
   struct RClass *sym;
 
-  mrb->symbol_class = sym = mrb_define_class(mrb, "Symbol", mrb->object_class);                 /* 15.2.11 */
+  mrb->symbol_class = sym = mrb_define_class(mrb, "Symbol", mrb->object_class);  /* 15.2.11 */
```

Comment:
These unnecessary space modifications are not ideal. They make change tracking difficult.
Next time, please separate space changes from the actual changes.